docs/deployment/openshift.md

```diff
@@ -19,6 +19,15 @@ $ oc project invenio
 
 Before deploying, you need to provide the credentials so the application can access the different services. Replace the credentials and configuration below with your own:
 
+**Invenio secret key**:
+
+  ```console
+ $ SECRET_KEY=$(openssl rand -hex 8)
+ $ oc create secret generic \
+   --from-literal="INVENIO_SECRET_KEY=$SECRET_KEY" \
+   invenio-secrets
```

Comment:
Question: This only needs to be done once, right? From then on, the same secret key is used (is saved for future use)?

Reply:
Yes, it is just that it is stored as a "secret".  So in a more "secure" (note the " hehe) place.

And it should be kept, actually if you change it mid way previous data is not readable (if I am correct, just tested and it is the case).